pkgs/by-name/ad/add-determinism/package.nix

```diff
@@ -0,0 +1,68 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+  zlib,
+  python3,
+  stdenv,
+  ...
```

Comment:
Using omitted args in callPackage args is not an expected usage. It may not work properly with overrides.